src/snowflake/cli/plugins/connection/commands.py

```diff
@@ -32,10 +32,19 @@
 from snowflake.connector import ProgrammingError
 from snowflake.connector.config_manager import CONFIG_MANAGER
 
-app = SnowTyper(
-    name="connection",
-    help="Manages connections to Snowflake.",
-)
+
+class ConnectionAppCreator(SnowTyperCreator):
+    def create_app(self) -> SnowTyper:
+        app = SnowTyper(
+            name="connection",
+            help="Manages connections to Snowflake.",
+        )
+        self.register_commands(app)
+        return app
```

Comment:
Can't we simplify if to something like
```py
app: SnowTyperCreator = SnowTyperCreator.create_app(SnowTyper(
            name="connection",
            help="Manages connections to Snowflake.",
        ))
```
do we really need a custom class for every command group?

Reply:
I could pass `**kwargs` from `create_app` to the constructor, but then I'd have to modify `add_init_command`-like procedures to work with `SnowTyperCreator` - I will check

Reply:
changed as discussed